source/initial-dns-seedlist-discovery/tests/README.rst

```diff
@@ -23,11 +23,13 @@ these tests::
   _mongodb._tcp.test3.test.build.10gen.cc.  86400  IN SRV  27017  localhost.build.10gen.cc.
   _mongodb._tcp.test5.test.build.10gen.cc.  86400  IN SRV  27017  localhost.build.10gen.cc.
   _mongodb._tcp.test6.test.build.10gen.cc.  86400  IN SRV  27017  localhost.build.10gen.cc.
+  _mongodb._tcp.test11.test.build.10gen.cc. 86400  IN SRV  27017  localhost.build.10gen.cc.
 
   Record                                    TTL    Class   Text
   test5.test.build.10gen.cc.                86400  IN TXT  "connectTimeoutMS=300000&socketTimeoutMS=300000"
   test6.test.build.10gen.cc.                86400  IN TXT  "connectTimeoutMS=200000"
   test6.test.build.10gen.cc.                86400  IN TXT  "socketTimeoutMS=200000"
+  test11.test.build.10gen.cc.               86400  IN TXT  "connectTime" "outMS=150000" "&socketT" "imeoutMS" "=" "250000"
```

Comment:
I'm glad we're adding this test. PyMongo's implementation won't currently pass it.

Reply:
Oh - how would it have dealt with it?

Reply:
It currently concatenates the strings from a single TXT record with &, just like the data from multiple TXT records. That seemed like the obvious thing to do, but I wasn't thinking about the length limitations on a single string.